packages/jest-environment/src/index.ts

```diff
@@ -302,11 +302,20 @@ export interface Jest {
    */
   unmock(moduleName: string): Jest;
   /**
-   * Instructs Jest to use fake versions of the standard timer functions.
+   * Instructs Jest to use fake versions of the global date, performance,
+   * time and timer APIs. Fake timers implementation is backed by
+   * [`@sinonjs/fake-timers`](https://github.com/sinonjs/fake-timers).
+   *
+   * @remarks
+   * Calling `jest.useFakeTimers()` once again in the same test file would reinstall
+   * fake timers using the provided options.
    */
-  useFakeTimers(implementation?: 'modern' | 'legacy'): Jest;
+  useFakeTimers(
```

Comment:
should we allow passing a string for the implementation? as shorthand (and would mean it's not a breaking change that people need to chamge their code for)